microsoft-edge/privacy-whitepaper/index.md

```diff
@@ -21,43 +21,100 @@ This whitepaper focuses on the desktop version of Microsoft Edge. Parts of the d
+1. Click the **Search suggestions and filters** section.  The **Search suggestions and filters** settings page<!-- edge://settings/searchFilters --> opens.
 
-Typed characters and the websites you visit are stored locally on the device per profile.  To delete the data, go to `edge://settings/clearBrowserData`. In the **Clear browsing data** window, select the **Browsing history** checkbox, and select the **Clear now** button.
+1. Use the toggle **Show me search and site suggestions using my typed characters**:
 
-If Bing is your default search provider and you are signed in, you may delete your searches through the [Microsoft Privacy dashboard](https://account.microsoft.com/privacy/).  You can clear your browsing history and delete websites from appearing as suggestions in the address bar. Go to `edge://history`, and select **Clear browsing data**.  You can delete the data Microsoft collects from the address bar and search suggestions features on Windows 10 and later. Open **Start** > **Settings** > **Privacy** > **Diagnostics & feedback**. In **Delete diagnostic data**, select **Delete**.  All other data is deleted after 36 months.
+   ![The toggle in Edge Settings for "Show me search and site suggestions using my typed characters"](./index-images/settings-and-more-icon.png)
 
-<!-- Deleting diagnostic data could use rewrite -->
+   If you turn off that toggle, your typed characters are no longer sent to your default search provider, but your search queries are still sent to your default search provider, to provide search results.<!--per ui: "If you turn this off, you'll only see suggestions from your favorites and history. Typed characters won't be sent to the search engine used in the address bar." -->
+
+
+<!-- ------------------------------ -->
+#### Enterprise Search provider
+
+If you're signed into Microsoft Edge using your work or school account (AAD account), your typed-in characters are also sent to the Microsoft Enterprise Search provider, which returns the work- or school-related content or cross-device suggested queries to you.  Your tenant admin can choose to turn on or off the work- or school-related content or query suggestions entirely for your organization, by using the `AddressBarMicrosoftSearchInBingProviderEnabled` MSB<!-- todo: "expand MSB" --> policy.  If your tenant admin choses to turn off that policy, you won't see the query suggestions from another device, and won't see all the work-related results from another device, but the query suggestions will still be returned from<!-- todo: from/to/in/by? --> your local browser,<!-- todo: change to "your current device's browser", or strike "local" -->, and results will be returned from your default search engine.
+
+You can further protect your privacy by doing any of the following:
```

Comment:
check lead-in against the orig draft.

orig draft:
Individually, you also have the flexibility to further protect your privacy by using either one of 3 options below:

Reply:
conclusion: Even if the first two are alternatives, it works, the way it's written.